src/test/java/net/minecraftforge/debug/entity/player/PlayerFindAmmoEventTest.java

```diff
@@ -0,0 +1,40 @@
+package net.minecraftforge.debug.entity.player;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.inventory.EnderChestInventory;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.event.entity.player.PlayerFindAmmoEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import org.apache.logging.log4j.Logger;
+
+@Mod("player_find_ammo_event_test")
+@Mod.EventBusSubscriber
+public class PlayerFindAmmoEventTest
+{
+    private static final boolean ENABLE = true;
+    private static final Logger logger = org.apache.logging.log4j.LogManager.getLogger(PlayerFindAmmoEvent.class);
+
+    @SubscribeEvent
+    public static void onPlayerFindAmmo(PlayerFindAmmoEvent event)
```

Comment:
The event says it is cancelable and needs to be canceled for it to apply. Why is it not happening here?

Reply:
I've added the cancellation and cancelled check now.